e2e-tests/tfm/cypress/e2e/journeys/utilisation-reports/fee-record-correction/feature-flag-enabled/create-fee-record-correction.spec.js

```diff
@@ -49,34 +55,46 @@ context('When fee record correction feature flag is enabled', () => {
   });
 
   context('PDC_RECONCILE users can create record correction requests', () => {
+    const { createFeeRecordCorrectionRequestPage } = pages;
+
     beforeEach(() => {
       pages.landingPage.visit();
       cy.login(USERS.PDC_RECONCILE);
 
       cy.visit(`utilisation-reports/${reportId}`);
     });
 
-    it('should display error message if invalid selections for creating fee record correction request', () => {
+    it('should be able to create a record correction request', () => {
+      premiumPaymentsTab.premiumPaymentsTable.checkbox([feeRecordAtToDoStatus.id], feeRecordAtToDoStatus.paymentCurrency, feeRecordAtToDoStatus.status).click();
+
       premiumPaymentsTab.createRecordCorrectionRequestButton().should('exist');
       premiumPaymentsTab.createRecordCorrectionRequestButton().click();
 
-      cy.url().should('eq', relative(`/utilisation-reports/${reportId}`));
+      cy.url().should('eq', relative(`/utilisation-reports/${reportId}/create-record-correction-request/${feeRecordAtToDoStatus.id}`));
 
-      errorSummary().contains('Select a record to create a record correction request');
-      premiumPaymentsTab.premiumPaymentsTable.error().should('exist');
-      cy.assertText(premiumPaymentsTab.premiumPaymentsTable.error(), 'Error: Select a record to create a record correction request');
-    });
+      //---------------------------------------------------------------
```

Comment:
The diff above has got slightly confusing - the error test case that was here has been removed as this is a duplicate of a test case in the existing premium payments error file (added prior to this PR)

Reply:
Do we need this line @sw-mattw or can it be moved, These two lines don't make sense:

```
Complete record correction request information form
```
- Text is then asserted rather than completing a form

```
Checks the information entered for record correction is correct
```
A URL is then asserted

Reply:
I've changed this to be paged-based naming (vs action) so the comments should make more sense now thanks